Neos.Neos/Tests/Behavior/Features/Bootstrap/FeatureContext.php

```diff
@@ -460,7 +460,7 @@ public function iWaitForElement($elementName)
      */
     public function locatePath($path)
     {
-        return parent::locatePath($this->resolvePath($path));
+        // return parent::locatePath($this->resolvePath($path));
```

Comment:
Is this intended?

Reply:
yes - these are the old MINK browser tests which are not run in CI and removed in 9.0 - but that this ended up here in the diff was a hickup in github ... now its gone ;)